app/main.py

```diff
@@ -1 +1,53 @@
-# write your code here
+class SoftwareEngineer:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = list()
+
+    def learn_skill(self, skill: str) -> None:
+        if skill not in self.skills:
+            self.skills.append(skill)
```

Comment:
Interesting idea - that skills should have only unique skills, that wasn't mentioned in task description, but if it was - it would be great to use `set` instead of `list` here - and then no additional check is required :)

Reply:
But no need to fix it here, because it doesn't mentioned in task description